target/linux/ramips/dts/mt7621_dual-q_h721.dts

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7621_dual-q_h721.dtsi"
+
+/ {
+	compatible = "dual-q,h721", "mediatek,mt7621-soc";
+	model = "Dual-Q H721";
+};
+
+&spi0 {
+	status = "okay";
+
+	flash@0 {
+		compatible = "jedec,spi-nor";
+		reg = <0>;
+		spi-max-frequency = <14000000>;
```

Comment:
Can this go faster? Try 40 or 50 MHz and compare the result with `time dd if=/dev/mtd3 of=/dev/null`.

Reply:
thanks， I compare

set spi-max-frequency = <14000000>：
root@OpenWrt:/# time dd if=/dev/mtd3 of=/dev/null
32128+0 records in
32128+0 records out
real    0m 11.95s
user    0m 0.05s
sys     0m 11.90s
root@OpenWrt:/# 


set spi-max-frequency = <40000000>：
root@OpenWrt:/# time dd if=/dev/mtd3 of=/dev/null
32128+0 records in
32128+0 records out
real    0m 5.29s
user    0m 0.00s
sys     0m 5.27s